sklearn/model_selection/_validation.py

```diff
@@ -1439,7 +1452,8 @@ def _translate_train_sizes(train_sizes, n_max_training_samples):
 
 
 def _incremental_fit_estimator(estimator, X, y, classes, train, test,
-                               train_sizes, scorer, verbose, return_times):
+                               train_sizes, scorer, verbose, return_times,
+                               fit_params):
```

Comment:
Is `fit_params` being used by this function?

Reply:
Removed this, turns out it was not being used.